services/bankid-api/serverless.yml

```diff
@@ -72,5 +72,5 @@ functions:
     events:
       - http:
           path: auth/bankid/cancel
-          method: delete
+          method: post
```

Comment:
why not delete?

Reply:
Becouse the BankID docs uses post, and since we reference ro that documentation.

Would also argue that we do not remove anything, it's an abort/cancel operation 🙂

Reply:
good point